buildkite/src/Jobs/Release/MinaArtifact.dhall

```diff
@@ -33,6 +33,7 @@ Pipeline.build
           S.exactly "buildkite/scripts/build-artifact" "sh",
           S.exactly "buildkite/scripts/connect-to-testnet-on-develop" "sh",
           S.exactly "dockerfiles/scripts/healthcheck-utilities" "sh",
+          S.exactly "buildkite/nightly" "txt",
```

Comment:
This works for sure though feels like the added dummy "nightly.txt" file could be replaced with the Dhall file itself.